bokehjs/src/coffee/models/glyphs/ray.coffee

```diff
@@ -1,10 +1,7 @@
-import {Glyph, GlyphView} from "./glyph"
+import {XYGlyph, XYGlyphView} from "./xy_glyph"
 import * as p from "../../core/properties"
 
-export class RayView extends GlyphView
-
-  _index_data: () ->
-    @_xy_index()
```

Comment:
It seems to me that this indexing for rays would be improved by taking the ray extent into account, but then we'd have to not make it an `XYGlyph` Maybe better just to leave it not an `XYGlyph`?